rwgame/debugstate.cpp

```diff
@@ -285,6 +290,34 @@ Menu* DebugState::createAIMenu()
 	return m;
 }
 
+Menu*DebugState::createWeaponMenu()
+{
+	CharacterObject* player = nullptr;
+	if (game->getPlayer()) {
```

Comment:
getPlayer() might be null, but if not, getCharacter() has to be called on it, and the result of which is used (and checked) in the lambda. It could all be done inside the lambda, but it might be unnecessary to do this vtable parkour in a loop. However, is the CharacterObject guaranteed to continue existing during the DebugState's lifetime? Otherwise the lambda might be poking at invalid memory?